demo/demo_secondary.py

```diff
@@ -499,10 +498,9 @@ def update_cycle():
 
   # 2. Set the fileinfo in the secondary_ecu object to the target info for the
   #    new firmware.
-  print("Before update", secondary_ecu)
   secondary_ecu.firmware_fileinfo = expected_target_info
-  secondary_ecu.update_release_counter(expected_target_info['fileinfo']['custom']['release_counter']) 
-  print("Updated", secondary_ecu)
+  secondary_ecu.update_release_counter(expected_target_info['fileinfo']\
```

Comment:
FYI, the `\` here is not necessary because you're inside parentheses. You can exclude that.

Minor tip, also: if you're already using an additional line and the full contents of the parentheses will fit on that line, that's slightly easier to read than splitting the contents across two lines. (I think [PEP 8](https://www.python.org/dev/peps/pep-0008/) also recommends something similar, though it's a little more finicky about the details.)

So in this case:

```Python
  secondary_ecu.update_release_counter(
      expected_target_info['fileinfo']['custom']['release_counter'])
```